index.html

```diff
@@ -0,0 +1,259 @@
+      <h4 class="main-program-header">Main Program</h4>
+      <div class="h_line"></div>
+      <div class="program-card-section">
+        <div class="main-program-card1">
+          <span class="material-symbols-outlined">
+            mark_unread_chat_alt
+          </span>
+          <p class="main-program-card-header">Sustainability</p>
+          <p class="main-program-card-desp">
+            Listen to welcome introduction by Agbabiaka Abdulquadri,
+            CEO|General Manager, Just Agric Company.
+          </p>
+        </div>
+        <div class="main-program-card1">
+          <span class="material-symbols-outlined">
+            satellite_alt
+          </span>
+          <p class="main-program-card-header">Improvement</p>
+          <p class="main-program-card-desp">
+            The Africa. agricultural value chain and consumers alike, rely on
+            producers.
+          </p>
+        </div>
+        <div class="main-program-card1">
+          <div class="first"><span class="material-symbols-outlined">people_outline</span></div>
+          <div class="second">
+            <p class="main-program-card-header">Habitabilities</p>
+          </div>
+          <div class="third">
+            <p class="main-program-card-desp">
+              Widespread positive environmental outcomes for the food and
+              agricultural industry? 
+            </p>
+          </div>
+        </div>
+        <div class="main-program-card1">
+          <span class="material-symbols-outlined">
+            bar_chart
+          </span>
+          <p class="main-program-card-header">Vaccinologists</p>
+          <p class="main-program-card-desp">
+            Listen to welcome introduction by Agbabiaka Abdulquadri,
+            CEO|General Manager, Just Agric Company.
+          </p>
+        </div>
+        <div class="main-program-card1">
+          <span class="material-symbols-outlined">
+            stream
+          </span>
+          <p class="main-program-card-header">Hagiographical</p>
+          <p class="main-program-card-desp">
+            In dairy and finishing beef production systems, feed not only
+            accounts.</p>
+          </p>
+        </div>
+      </div>
+      <div class="see-button">
+        <a href="#" class="see-more">SEE THE WHOLE PROGRAM</a>
+        <div class="see"></div>
+        <button class="main-program-button"><a href="#">Join Agriculture Summit 2023</a> </button>
+      </div>
+    </section>
+
+    <section id="feature-section"></section>
+    <section id="feature-speakers">
+      <div class="h3-line">
+        <h3 class="speaker-h3">Feature Speakers</h3>
+        <div class="h_line"></div>
+      </div>
+      <div class="card-wrapper">
+        <!-- <div class="test">
+        <div id="card1">
+          <div class="dot-photo">
+            <img src="./images/rectdot.png" alt="" class="dot">
+            <img src="./images/Abebe-Menkir.jpg" alt="" class="speaker-img">
+          </div>
```

Comment:
- [ ] Please consider removing `commented codes` as it will make your codebase messy, Remember if you ever like to see the codes you have written once you can use your commit history.